dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -10,19 +10,123 @@
+# The URI parameter in these messages is for the ACME certificate authority, such as https://sampleCAProvider.org/directory
 
-#-----  ACME messages
+CWPKI2001E=CWPKI2001E: The ACME certificate authority at the {0} URI responded that the authorization challenge failed for the {1} domain. The challenge status is {2}.  The error is {3}.
+CWPKI2001E.explanation=The challenge status indicated that the authorization challenge request failed and a certificate cannot be created.
+CWPKI2001E.useraction=Review the status message and error for details on the failure.
+
+CWPKI2002E=CWPKI2002E: Polling for a successful authorization challenge for the {0} domain from the ACME certificate authority at the {1} URI timed out. The status is {2}. The configured timeout is {3}.
+CWPKI2002E.explanation=The certificate authority challenge request was not validated in the configured time and a certificate cannot be created.
+CWPKI2002E.useraction=Review the certificate authority URI provided. Verify that the URI can be successfully accessed by the calling server. Verify that the calling server can receive a response from the certificate authority. Review the status code. Some certificate authorities might require a longer timeout.
+
+CWPKI2003E=CWPKI2003E: The ACME certificate authority at the {0} URI responded that the certificate order failed for the {1} domains. The order status is {2}. The error is {3}.
+CWPKI2003E.explanation=The order status indicated that the authorization order request failed and a certificate cannot be created.
+CWPKI2003E.useraction=Review the status message and error for details on the failure.
+
+CWPKI2004E=CWPKI2004E: Polling for a successful order for the {0} domain from the ACME certificate authority at the {1} URI timed out. The status is {2}.  The configured timeout is {3}.
+CWPKI2004E.explanation=The certificate authority domain certificate order request did not complete in the configured time and a certificate cannot be created.
+CWPKI2004E.useraction=Review the certificate authority URI provided. Verify that the URI can be successfully accessed by the calling server. Verify that the calling server can receive a response from the certificate authority. Review the status code. Some certificate authorities might require a longer timeout.
+
+CWPKI2005E=CWPKI2005E: The authorization returned from the ACME certificate authority at the {0} URI did not include a valid challenge type. Supported challenge types include {1}.
+CWPKI2005E.explanation=The certificate authority returned a challenge type that is currently unsupported. Verify that the certificate authority uses a type in the supported list.
+CWPKI2005E.useraction=Select a certificate authority that provides a supported challenge type.
+
+CWPKI2006I=CWPKI2006I: The ACME certificate authority at the {0} URI provided the following terms of service: {1}
+CWPKI2006I.explanation=The certificate authority provides terms of service. Review the terms of service provided.
```

Comment:
**change**
Review the terms of service provided.
**to**
Review the provided terms of service.

Is this needed? It's technically a useraction, though I know informational messages don't have useractions